content/en/docs/concepts/policy/pid-limiting.md

```diff
@@ -0,0 +1,43 @@
+---
+reviewers:
+- derekwaynecarr
+title: PID Limiting
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
+
+PID Limiting features allow administrators are able to provide pod-to-pod PID isolation by defaulting the number of PIDs per pod. In addition, administrators can enable node-to-pod PID isolation by reserving a number of allocatable PIDs to user pods via node allocatable.
+
+<!-- body -->
+
+Process IDs (PIDs) are a fundamental resource on Linux hosts. It is trivial to hit the task limit without hitting any other resource limits and cause instability to a host machine. Administrators require mechanisms to ensure that user pods cannot induce PID exhaustion that prevents host daemons (runtime, kubelet, etc) from running. In addition, it is important to ensure that PIDs are limited among pods in order to ensure they have limited impact to other workloads on the node.
+
+You can configure a kubelet to limit the number of PIDs a given pod can consume. If that machine supports `32,768` PIDs and `100` pods, one can give each pod a budget of `300` PIDs to prevent total exhaustion of PIDs. If the admin wants to overcommit PIDs similar to cpu or memory, they may do so as well with some additional risks. Either way, no one pod can bring the whole machine down. This will generally prevent against simple fork bombs from taking over your cluster.
+
+Per-pod PID limiting allows administrators to protect one pod from another, but does not ensure if all pods on the machine can protect the node, and the node agents themselves from falling over.
+
+Node PID limiting provides isolation of PIDs from end user workloads on a pod from the node agents (kubelet, runtime, etc.). The admin is able to reserve a specific number of PIDs - similar to how one reserves CPU or memory today - and ensure they are never consumed by pods on that machine.
+
+PID limiting is a an important part of the family of [compute resources](/docs/concepts/configuration/manage-resources-containers/) requests and limits.
```

Comment:
```suggestion
PID limiting is a an important sibling to [compute resource](/docs/concepts/configuration/manage-resources-containers/) requests and limits. However, you specify it in a different way: rather than defining a Pod's resource limit in the `.spec` for a Pod, you configure the limit as a setting on the kubelet.

{{< caution >}}
This means that the limit that applies to a Pod may be different depending on where the Pod is scheduled. To make things simple, it's easiest if all Nodes use the same PID resource limits and reservations.
{{< /caution >}}
```